src/Simplex/Messaging/Agent/Store/SQLite.hs

```diff
@@ -281,6 +283,48 @@ instance (MonadUnliftIO m, MonadError StoreError m) => MonadAgentStore SQLiteSto
   getMsg :: SQLiteStore -> ConnAlias -> InternalId -> m Msg
   getMsg _st _connAlias _id = throwError SENotImplemented
 
+  createBcast :: SQLiteStore -> BroadcastId -> m ()
+  createBcast SQLiteStore {dbConn} bId =
+    liftIOEither $
+      checkConstraint SEBcastDuplicate $
+        DB.execute dbConn "INSERT INTO broadcasts (broadcast_id) VALUES (?);" (Only bId)
```

Comment:
please let's stick to named parameters here and in other bcast methods. I understand it's only 1 here, but I'd rather be consistent with the rest of the code than save a few symbols

Reply:
I suggest we have up to three without naming them, as long as they as sufficiently close - it's equivalent to parameters in the functions. Otherwise the same parameter has to be named 3 times (2 times as query param and 1 as a name in code).